maps/site53/items/clothing/solgov-under.dm

```diff
@@ -226,7 +226,7 @@
 
 /obj/item/clothing/under/solgov/utility/fleet
 	name = "utility coveralls"
-	desc = "The utility uniform of the SCP Foundation, made from an insulated material."
```

Comment:
we are in the northern hemisphere and outside of the facility visibly contains snow
not sure why you'd change that line

Reply:
[Alaska is in both hemispheres](https://en.wikipedia.org/wiki/Geography_of_Alaska)